spring-content-rest/src/main/java/internal/org/springframework/content/rest/controllers/resolvers/DefaultEntityResolver.java

```diff
@@ -174,11 +174,13 @@ public Object findOne(Repositories repositories, StoreInfo info, Class<?> domain
                     domainObj = invoker.invokeFindById(id);
                 }
             } catch (ConverterNotFoundException e) {
-
+                logger.debug("resolveRootResourceInformation failed with root resource information class present");
                 domainObj = findOneByReflection(repositories, domainObjClass, id);
+            } catch (AccessDeniedException ace) {
```

Comment:
catching the ACE explicitly would be good for code clarity. Regardless if you want to throw it or not.